tests/integration/drivers/vector/test_astra_db_vector_store_driver.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+import json
+import math
+import os
+
+import pytest
+
+from griptape.drivers import AstraDBVectorStoreDriver, BaseVectorStoreDriver
+from tests.mocks.mock_embedding_driver import MockEmbeddingDriver
+
+TEST_COLLECTION_NAME = "gt_int_test"
+TEST_COLLECTION_NAME_METRIC = "gt_int_test_dot"
+
+
+class TestAstraDBVectorStoreDriver:
+    def _descore_entry(self, entry: BaseVectorStoreDriver.Entry) -> BaseVectorStoreDriver.Entry:
+        return BaseVectorStoreDriver.Entry.from_dict({k: v for k, v in entry.__dict__.items() if k != "score"})
```

Comment:
Please move protected methods after public methods (sorry we don't have an [linter rule](https://github.com/astral-sh/ruff/issues/2425) for this yet)

Reply:
Sure, no problem.